src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,44 @@
+.calendar {
+  display: flex;
+  margin: 0 auto;
+  width: 7 * $square-size + 6 * $square-gap + 2 * $square-paddings;
```

Comment:
just specify `box-sizing`

Reply:
sorry, can you explain that in more detail?

Reply:
Hey, you can tag mentor in fe_chat and ask him about this comment.
I see that you have already added `box-sizing` rule to * selector and everything looks good to me.

Reply:
Thanks!)